### DISCUSSION TITLE
postCreateCommand does not run

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#26379] [Codespaces]
<p>I have noticed that the <code>postCreateCommand</code> config in my <code>devcontainer.json</code> does not get honored any more and does not run.</p>
<p>Not a huge problem, but it means I have run <code>yarn</code> manually before I can use a codespace (Node project).</p>

**Comment by osortega** [comment]
<p>Hey! That is very strange.<br>
Is the repository you are using public so I can try to reproduce this?<br>
I tried with a couple different ones and they seem to be working fine.</p>

**Comment by maiertech** [comment]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/maiertech/gatsby-themes" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/1482402/181097402-99e9c43e-29fe-4276-be74-e9df45c17084.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/maiertech/gatsby-themes" target="_blank" rel="noopener nofollow ugc">maiertech/gatsby-themes</a></h3>

<p>Gatsby themes for @maiertech and @454de6e websites. - maiertech/gatsby-themes</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by osortega** [comment]
<p>I see it running <code>yarn install</code> as the postCreateCommand.<br>
I think what seems to be happening is that we are not showing the toast saying that we are still running things in the background. That seems to be a bug.</p>
<p>If I manually open the Creation Log I see <code>yarn install</code> output.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/8/d86ac2e94d7fb9f87bfef66590506d264db34219.png" data-download-href="/uploads/short-url/uSvRRFXpVXgaY3v3lDEy7Joll7r.png?dl=1" title="Screen Shot 2020-11-10 at 2.04.24 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/48293249/181097406-e1fa4f7d-1eb7-4e0d-9e4a-c6e49e28877e.png" alt="Screen Shot 2020-11-10 at 2.04.24 PM" data-base62-sha1="uSvRRFXpVXgaY3v3lDEy7Joll7r" width="689" height="272" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d86ac2e94d7fb9f87bfef66590506d264db34219_2_689x272.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d86ac2e94d7fb9f87bfef66590506d264db34219_2_1033x408.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d86ac2e94d7fb9f87bfef66590506d264db34219_2_1378x544.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d86ac2e94d7fb9f87bfef66590506d264db34219_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-11-10 at 2.04.24 PM</span><span class="informations">2586×1020 203 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Could you help confirming if this is true on your side as well?</p>

**Comment by maiertech** [comment, marked as answer]
<p>This seems to work now.</p>
